content/en/docs/reference/setup-tools/kubeadm/running-unit-tests.md

```diff
@@ -0,0 +1,36 @@
+---
+approvers:
+- luxas
+- timothysc
+- fabriziopandini
+title: Running unit tests
```

Comment:
I would turn this into contributing.md instead, and include more content from https://github.com/kubernetes/kubeadm/blob/master/docs/release-cycle.md. Can you do that please?

Reply:
ok, i can try.

Reply:
My rough feeling is that this doc should go https://github.com/kubernetes/kubeadm/tree/master/docs, not here.

Website doest have a contribute.md file (https://kubernetes.io/docs/home/contribute/), however, it is completely about how to contribute to docs.

If we agree that "contributing to kubernetes (including its subprojects)" needs to be documented here, we will have to dump a lot of developer knowledge.

Reply:
@luxas, i agree with @tengqm .

what do you think?
i can add this doc under https://github.com/kubernetes/kubeadm/tree/master/docs an call it `contributing.md`

the user is probably going to use google anyhow for this. for instance googling `kubeadm release cycle` returns exactly the page of interest.
https://github.com/kubernetes/kubeadm/blob/master/docs/release-cycle.md

googling `kubeadm unit tests` would work too.